src/api/app/controllers/webui/webui_controller.rb

```diff
@@ -38,8 +38,11 @@ def handle_unverified_request
        when "branch?" then "branch"
        else exception.query
     end
-    message = "Sorry, you are not authorized to " +
-              (exception.message ? "#{pundit_action} this #{exception.record.class}." : "perform this action.")
+    if pundit_action && exception.record
+      message = "Sorry, you are not authorized to #{pundit_action} this #{exception.record.class}."
```

Comment:
This is not about `NoMethodError` it's only about authorization.

Trying again will not result in any difference so it's really nasty advise ;-) The more detailed the error message the easier it is for people to think about the reason why this was not authorized and hence come up with a solution themselves.